cardano-testnet/src/Testnet/Start/Cardano.hs

```diff
@@ -144,7 +144,7 @@ getDefaultShelleyGenesis asbe maxSupply opts = do
 -- > │   └── README.md
 -- > ├── drep-keys
 -- > │   ├── drep{1,2,3}
--- > │   │   └── drep.{skey,drep.vkey}
+-- > │   │   └── drep.{skey,vkey}
```

Comment:
How did you notice to change this?

Reply:
It's @carbolymer who did this change, I think after seeing my (late) review on https://github.com/IntersectMBO/cardano-node/pull/6007 :wink: